inc/metadata/namespace.php

```diff
@@ -0,0 +1,315 @@
+		case 'cc-by-nc':
+			$url = 'https://creativecommons.org/licenses/by-nc/4.0/';
+			break;
+		case 'cc-by-nc-sa':
+			$url = 'https://creativecommons.org/licenses/by-nc-sa/4.0/';
+			break;
+		case 'cc-by-nc-nd':
+			$url = 'https://creativecommons.org/licenses/by-nc-nd/4.0/';
+			break;
+		case 'all-rights-reserved':
+			$url = 'https://choosealicense.com/no-license/';
+			break;
+		default:
+			$url = 'https://choosealicense.com/no-license/';
+	}
+
+	return $url;
+}
+
+/**
+ * @param \WP_Post $post
+ */
+function add_expanded_metadata_box( $post ) {
+
+	if ( $post->post_type !== 'metadata' ) {
+		return;
+	}
+
+	if ( isset( $_GET['pressbooks_show_expanded_metadata'] ) && check_admin_referer( 'pb-expanded-metadata' ) ) {
+		update_option( 'pressbooks_show_expanded_metadata', $_GET['pressbooks_show_expanded_metadata'] );
+	}
+
+	$show_expanded_metadata = show_expanded_metadata();
+
+	$url = get_edit_post_link( $post->ID );
+	if ( $show_expanded_metadata ) {
+		$text = __( 'Hide Additional Book Information', 'pressbooks' );
+		$href = wp_nonce_url( $url . '&pressbooks_show_expanded_metadata=0', 'pb-expanded-metadata' );
+	} else {
+		$text = __( 'Show Additional Book Information', 'pressbooks' );
+		$href = wp_nonce_url( $url . '&pressbooks_show_expanded_metadata=1', 'pb-expanded-metadata' );
+	}
+
+	?>
+	<div id="expanded-metadata-panel" class="postbox">
+		<div class="inside">
+			<p><?php _e( 'The book information you enter here appears on your book’s cover and title pages and in the metadata of your webbook and exported files.', 'pressbooks' ); ?></p>
+			<?php if ( ! $show_expanded_metadata && $show_expanded_metadata !== null ) { ?>
+				<p><?php _e( 'If you need to enter additional information, click the button below to see all available fields.', 'pressbooks' ); ?></p>
+			<?php } ?>
+			<p><a class="button" href="<?php echo $href; ?>"><?php echo $text; ?></a></p>
+		</div>
+	</div>
+	<?php
+}
+
+/**
+ * Should we show expanded metadata fields or not?
+ *
+ * @return bool | null
+ */
+function show_expanded_metadata() {
+	if ( isset( $_GET['pressbooks_show_expanded_metadata'] ) && check_admin_referer( 'pb-expanded-metadata' ) && ! empty( $_GET['pressbooks_show_expanded_metadata'] ) ) {
+		return true;
+	} elseif ( ! empty( get_option( 'pressbooks_show_expanded_metadata' ) ) ) {
+		return true;
+	} else {
+		$metadata = Book::getBookInformation();
+		$additional_fields = [
+			'pb_author_file_as',
+			'pb_onsale_date',
+			'pb_copyright_year',
+			'pb_series_title',
+			'pb_series_number',
+			'pb_keywords_tags',
+			'pb_hashtag',
+			'pb_list_price_print',
+			'pb_list_price_pdf',
+			'pb_list_price_epub',
+			'pb_list_price_web',
+			'pb_audience',
+			'pb_bisac_subject',
+			'pb_bisac_regional_theme',
+			'pb_catalogue_order',
+		];
+		foreach ( $additional_fields as $field ) {
+			if ( isset( $metadata[ $field ] ) && ! empty( $metadata[ $field ] ) ) {
+				update_option( 'pressbooks_show_expanded_metadata', 1 );
+				return true;
```

Comment:
If a user sets `pb_list_price_epub`, then sets `$_GET['pressbooks_show_expanded_metadata'] = 0`, then reloads the Book Information page, we end up in this code block and return true.

Is this intentional?